docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: '3'
```

Comment:
What do you think about putting docker stuff in a `.docker-tooling` directory like we do in the socialbro repository?